ansible/group_vars/all

```diff
@@ -48,3 +48,40 @@ db:
   authkeys:
   - guest
   - whisk.system
+
+# The default value for catalog_source is empty, meaning that
+# openwhisk will not install the catalog from the catalog repositories.
+# If catalog_source is not set to "catalog-repos", the other variables
+# like namespace, api_host, auth_key_file and catalog_repos will not
+# be effective.
+catalog_source: ""
+
+# The default name space is /whisk.system. The namespace must begin with a slash "/".
+namespace: "/whisk.system"
+
+# The api_host is used to specify the IP address of the host, where
+# openwhisk edge host is running.
+api_host: "172.17.0.1"
+
+# The auth_key_file is used to determine where the authentication key is saved.
+# By default, we take the key from {{ openwhisk_home }}/config/keys/auth.whisk.system.
+auth_key_file: "{{ openwhisk_home }}/config/keys/auth.whisk.system"
```

Comment:
this is the key for installing the catalog, as such suggest we call it catalog_authkey.